java/com/google/flatbuffers/Table.java

```diff
@@ -72,9 +72,16 @@ protected static int __offset(int vtable_offset, int offset, ByteBuffer bb) {
    * @return Returns the relative offset stored at `offset`.
    */
   protected int __indirect(int offset) {
-    return offset + bb.getInt(offset);
+    return __indirect(offset, bb);
```

Comment:
A static version of the __indirect method has been added to the Table class for use in vectors.
The instance method calls a static method to avoid code duplication.
Leave duplication?

Reply:
Since `__indirect` is so core to FlatBuffers performance on Java, I'd say leave duplication to be safe, yes.

Reply:
Done